jnigen/example/in_app_java/jnigen.yaml

```diff
@@ -14,3 +14,5 @@ source_path:
 classes:
   - 'com.example.in_app_java.AndroidUtils' # from source_path
   - 'android.os.Build' # from gradle's compile classpath
+  - 'java.util.HashMap' # from gradle's compile classpath
```

Comment:
Should we add a separate test instead of adding it on to an example?

If we release the standard library bindings as a separate package, we would likely want to remove HashMap from here again for example purposes. We shouldn't lose test coverage when doing that.

Maybe we should cover tests in `test/` and only add things to `example/` when we really want to illustrate something specific. And then it should probably be accompanied by a README.md detailing what the example shows.

Reply:
Good idea to separate the example. Will do that.